e2etests-cli/main_test.go

```diff
@@ -2,20 +2,86 @@ package e2etests_cli
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 	"testing"
 	"time"
+
+	"oss.terrastruct.com/d2/d2cli"
+	"oss.terrastruct.com/util-go/assert"
+	"oss.terrastruct.com/util-go/xmain"
+	"oss.terrastruct.com/util-go/xos"
 )
 
 func TestCLI_E2E(t *testing.T) {
 	t.Parallel()
 
 	tca := []struct {
 		name string
-		run  func(t *testing.T, ctx context.Context)
+		run  func(t *testing.T, ctx context.Context, dir string, env *xos.Env)
 	}{
 		{
-			name: "hello_world",
-			run:  func(t *testing.T, ctx context.Context) {},
+			name: "hello_world_png",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "hello-world.d2", `x -> y`)
+				err := runTestMain(t, ctx, dir, env, "hello-world.d2", "hello-world.png")
+				assert.Success(t, err)
+				png := readFile(t, dir, "hello-world.png")
+				assert.Testdata(t, ".png", png)
+			},
+		},
+		{
+			name: "hello_world_png_pad",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "hello-world.d2", `x -> y`)
+				err := runTestMain(t, ctx, dir, env, "--pad=400", "hello-world.d2", "hello-world.png")
+				assert.Success(t, err)
+				png := readFile(t, dir, "hello-world.png")
+				assert.Testdata(t, ".png", png)
+			},
+		},
+		{
+			name: "hello_world_png_sketch",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "hello-world.d2", `x -> y`)
+				err := runTestMain(t, ctx, dir, env, "--sketch", "hello-world.d2", "hello-world.png")
+				assert.Success(t, err)
+				png := readFile(t, dir, "hello-world.png")
+				assert.Testdata(t, ".png", png)
+			},
+		},
+		{
+			name: "multiboard/life",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "life/index.d2", `x -> y
```

Comment:
why does this need to be specified like that? does `life.d2` not work?

Reply:
It doesn't, I added this just now to test if you point to a directory, `d2` will automatically pick up the `index.d2`. I can add another multiboard to test what you're asking.